addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -660,7 +660,7 @@ odoo.define('website.s_website_form', function (require) {
                 }
 
                 const formData = new FormData(this.$target[0]);
-                const currentValueOfDependency = formData.get(dependencyName);
+                const currentValueOfDependency = ["contains","!contains"].includes(comparator)?formData.getAll(dependencyName).join():formData.get(dependencyName);
```

Comment:
The line is still too long for no reason, making it hard to read.
It also contains many linter error, ideally try to have one configured please.

Could you just copy paste the code I provided you here https://github.com/odoo/odoo/pull/125521#discussion_r1265640862 ?
Or at least do the same formatting by splitting your ternary on multi line

Reply:
@rdeodoo I am so sorry for that. I have made the changes now